GW2EIEvtcParser/EIData/ProfHelpers/Thief/ThiefHelper.cs

```diff
@@ -142,12 +144,13 @@ internal static void ComputeProfessionCombatReplayActors(AbstractPlayer player,
                 {
                     (int, int) lifespan = ProfHelper.ComputeEffectLifespan(log, exit, 8000, player.AgentItem, ShadowPortalOpenedBuff);
                     var connector = new PositionConnector(exit.Position);
-                    replay.Decorations.Add(new CircleDecoration(true, 0, 90, lifespan, color.WithAlpha(0.5f).ToString(), connector).UsingSkillMode(player, Spec.Thief, ShadowPortal, GenericAttachedDecoration.SkillModeCategory.Portal));
-                    GenericAttachedDecoration icon = new IconDecoration(ParserIcons.PortalShadowPortalOpen, CombatReplaySkillDefaultSizeInPixel, CombatReplaySkillDefaultSizeInWorld, 0.7f, lifespan, connector).UsingSkillMode(player, Spec.Thief, ShadowPortal, GenericAttachedDecoration.SkillModeCategory.Portal);
+                    var skill = new SkillModeDescriptor(player, Spec.Thief, ShadowPortal, SkillModeCategory.Portal);
+                    replay.Decorations.Add(new CircleDecoration(true, 0, 90, lifespan, color.WithAlpha(0.5f).ToString(), connector).UsingSkillMode(shadowPortalSkill));
```

Comment:
I'm guessing "shadowPortalSkill" here and the two next ones should be "skill"? On current the descriptor points towards "ShadowPortal" not "PrepareShadowPortal".